.circleci/config.yml

```diff
@@ -60,13 +64,17 @@ jobs:
     steps:
       - checkout
       - npm-install
+      - run: npm run build
       - run: npm run e2e:apib
 
   test-e2e-openapi2:
     docker: [<<: *node12]
     steps:
       - checkout
+      - restore_cache:
+          key: build-{{ .Branch }}-{{ .Revision }}-node12
```

Comment:
It might be worth updating `node12` string to use environment variable indicating the node version. I'm wondering about someone changing the node12 version somewhere and missing parts in the rest of the configuration file.

Reply:
Good catch. Also in the fact that this cache restoration is unnecessary. I've used it before when I wanted to store the build artifacts in cache, but after discussion with you I've removed any kind of build caching and perform build on each test job.

I will remove that `restore_cache` at all.

Reply:
Removed at fd46b03.